tsfresh/feature_extraction/settings.py

```diff
@@ -117,7 +119,7 @@ def __init__(self):
             "ar_coefficient": [{"coeff": coeff, "k": k} for coeff in range(5) for k in [10]],
             "mean_abs_change_quantiles": [{"ql": ql, "qh": qh}
                                           for ql in [0., .2, .4, .6, .8] for qh in [.2, .4, .6, .8, 1.]],
-            "fft_coefficient": [{"coeff": coeff} for coeff in range(0, 10)],
+            "fft_coefficient": [{"coeff": k, "attr": a} for a, k in product(["real", "imag"], range(100))],
```

Comment:
That would be 20x as many features from this calculator. That seems quite a lot for the default settings. How likely are they gonna be `NaN` anyway?